### PR TITLE
Fixes anomaly research ruin failing compile maps

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

There was a group of invalid keys which resulted in blank tiles, this was easy enough to fix.
## Why It's Good For The Game

I'd imagine prs passing linters and compile maps is a little important.
## Changelog
:cl:
fix: The anomaly research ruin is no longer missing several tiles on the inside to the void
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/anomaly_research.dmm [hunk 1]
```diff
@@ -1451,6 +1456,8 @@
 "MJ" = (
 /turf/closed/wall/r_wall,
 /area/misc/anomaly_research)
+"MM" = (
+)
```
Comment (san7890) [inline _maps/RandomRuins/SpaceRuins/anomaly_research.dmm:1460]: you just added a new one

Comment (san7890) [reply _maps/RandomRuins/SpaceRuins/anomaly_research.dmm:1460]: what the hell is wrong with this ruin man

**Review by san7890 — CHANGES_REQUESTED** [review]

File: _maps/RandomRuins/SpaceRuins/anomaly_research.dmm (hunk 1 above)
Comment (Paxilmaniac) [reply _maps/RandomRuins/SpaceRuins/anomaly_research.dmm:1460]: Whar the hell

Comment (Paxilmaniac) [reply _maps/RandomRuins/SpaceRuins/anomaly_research.dmm:1460]: Oh I missed a few spots thats why

**Pull request merged**